nuxeo-features/nuxeo-search/nuxeo-search-ui/src/main/java/org/nuxeo/search/ui/seam/SearchUIActions.java

```diff
@@ -270,7 +270,10 @@ public void setCurrentSelectedSavedSearchId(String selectedSavedSearchId) throws
     protected void resetCurrentContentViewWorkingList() {
         if (currentContentViewName != null) {
             ContentView contentView = contentViewActions.getContentView(currentContentViewName);
-            documentsListsManager.resetWorkingList(contentView.getSelectionListName());
+            if (contentView != null) {
+                String selectionListName = contentView.getSelectionListName();
```

Comment:
True, I could have put that in one line like documentsListsManager.resetWorkingList(contentView.getSelectionListName());
Should I correct the commit or just remember it for the next time ?